chroot-etc/intel-geteuid.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/src/intel_device.c b/src/intel_device.c
+index 140e153..56ef7ef 100644
+--- a/src/intel_device.c
++++ b/src/intel_device.c
+@@ -579,7 +579,7 @@ int intel_open_device(int entity_num,
+ 		master_count++;
+ 
+ 	/* Non-root user holding MASTER, don't let go */
+-	if (geteuid() && is_master(fd))
++	if (getuid0() && is_master(fd))
```

Comment:
crouton invented the function getuid0 (which just returns 0), just because it's the same length as "geteuid" and is thus easy to binary hack.  But really, this patch should just convert geteuid() to the constant 0 and let the compiler optimize the rest out...